src/Language/Haskell/Liquid/GHC/Interface.hs

```diff
@@ -143,8 +143,10 @@ configureDynFlags cfg tmp = do
                                                 (ModRenaming True [])
                                 : packageFlags df'
                  -- , profAuto     = ProfAutoCalls
-                 , ghcLink      = LinkInMemory
-                 , hscTarget    = HscInterpreted
+                 -- , ghcLink      = LinkInMemory
+                 -- , hscTarget    = HscInterpreted
+                 , ghcLink      = NoLink
+                 , hscTarget    = HscNothing
```

Comment:
@ranjitjhala it's not quite ready because of this change. Another issue Sean has run into is that base uses unboxed tuples, which ghci (and thus LH via `hscTarget=HscInterpreted` does not support).

Using `HscInterpreted` has always felt wrong to me since we don't actually need to do code generation, but switching to `HscNothing` causes issues for name resolution as it affects which / how many names GHC keeps in its environment after type checking...

I'm trying to fix the name resolution to work with `HscNothing`, but got sidetracked by drafting the dissertation. I'll take another look at it this afternoon.